test/built-ins/Promise/get-prototype-abrupt.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2019 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-promise-executor
+description: >
+  Promise constructor rethrows errors raised at getting prototype.
```

Comment:
`rethrows` is not conventionally correct, maybe just `return abrupt completions`, this might be fixed in a follow up

Reply:
#2174

Reply:
that was fast, thanks!